Userland/Games/Solitaire/main.cpp

```diff
@@ -177,6 +177,12 @@ int main(int argc, char** argv)
         game.setup(mode);
     }));
     game_menu.add_separator();
+    NonnullRefPtr<GUI::Action> undo_action = GUI::Action::create("&Undo", { Mod_None, Key_U }, [&](auto&) {
+        game.undo_perform();
+    });
```

Comment:
Please use `GUI::CommonActions::make_undo_action()` which sets the default key combination and also has a nice icon : )

Reply:
Thanks! Wasn't aware of `CommonActions`, very nice.